src/tools/dart-run.md

```diff
@@ -129,13 +129,30 @@ $ dart run foo arg1 arg2
 ## Debugging
 
 To enable debugging, pass one or more debugging options.
-Here's an example of enabling [`assert` statements][assert]:
+
+To enable [`assert` statements][assert],
+specify the `--enable-asserts` flag:
 
 ```terminal
 $ dart run --enable-asserts tool/debug.dart
 ```
 
-See `dart run --help` for details.
+To enable debugging through [Dart DevTools](/tools/dart-devtools),
+a set of debugging and performance tools,
+specify the `--observe` flag.
```

Comment:
**issue:** These are a series of non-exclusive options. Consider a list here.
```suggestion
- To enable [`assert` statements][assert],
add the `--enable-asserts` flag.

  \```terminal
  $ dart run --enable-asserts tool/debug.dart
  \```

- To enable debugging through [Dart DevTools](/tools/dart-devtools)
  add the `--observe` flag.
```

Reply:
I was trying to structure this section more as a mini-tutorial rather than as a reference as we don't include all the flags here, particularly since it usually makes sense to use `--observe` and `--pause-isolates-on-start` together. If we want to adjust to this more reference style, how should we keep that association without requiring users to refer to multiple different parts.

Reply:
**comment:** If you don't want to provide all debugging flags here (and there's no reason to do so), then modify the intro paragraph. I'll make a note there.

You can also add a graf at the end that says:
```
To learn more about debugging options, see [Dart debugging](link).
```